### ISSUE TITLE
Webpack 5 support

### ISSUE THREAD

**Issue by ScriptedAlchemy** [sysgears/webpack-virtual-modules#43]
Any hints on what I need to PR to get Webpack 5 support?

**Comment by larixer** [comment]
I have no idea to be fair. Webpack 5 is very different and I haven't seen good documentation for Webpack 5 internals

**Comment by ScriptedAlchemy** [comment]
Checking the filesystem API of Webpack Core, looks like we are good. 

Testing it now, seems to keep on working. Might ping you with a question or two. Im using it to workaround another issue in the webpack core that we haven't yet solved. In conjunction with https://github.com/webpack/webpack/issues/10352

**Comment by larixer** [comment]
@ScriptedAlchemy Sure, feel free to ping me with questions

**Comment by ScriptedAlchemy** [comment]
https://youtu.be/uFZsdShuzxU did a screencast on it :)

**Comment by larixer** [comment]
@ScriptedAlchemy Cool, thank you for sharing!

**Comment by kirill-konshin** [comment]
Got an error:

```
TypeError: Cannot read property 'data' of undefined
ERR!     at setData (/Users/xxx/.yarn/cache/webpack-virtual-modules-npm-0.2.2-1f43412872-f06d8f3e84.zip/node_modules/webpack-virtual-modules/index.js:84:15)
```

**Comment by larixer** [comment]
@kirill-konshin Because webpack 5 is not supported yet

**Comment by kirill-konshin** [comment]
Why is this issue closed then :) let's leave it open

**Comment by larixer** [comment]
@kirill-konshin Because issue author closed it

**Comment by larixer** [comment]
@ScriptedAlchemy @kirill-konshin Fixed via #55 and published in `webpack-virtual-modules@0.3.0`
